packages/video_player/video_player/example/ios/Runner/Info.plist

```diff
@@ -27,6 +29,10 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>audio</string>
```

Comment:
is this setting required for this PR to work? Note that it's part of the example project.